core/filters/test/unit/date.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { date } from '@vue-storefront/core/filters/date'
+
+jest.mock('@vue-storefront/core/helpers', () => ({
+  once: jest.fn()
```

Comment:
Maybe if you would try `  once: jest.fn((a, b) => b())` to invoke `dayjs.extend(dayjsLocalizedFormat)` it would be possible to achieve 100% cov?